spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -38,3 +38,38 @@ A full list of changes may be found on the link:https://github.com/eclipse/micro
 === Specification Changes
 
 - Clarified what it means for a value to be present (link:https://github.com/eclipse/microprofile-config/issues/216[#216])
+
+== Release Notes for MicroProfile Config 1.2
+
+The following changes occurred in the 1.2 release, compared to 1.1
+
+A full list of changes may be found on the link:https://github.com/eclipse/microprofile-config/milestone/3?closed=1[MicroProfile Config 1.2 Milestone]
+
+=== API/SPI Changes
+
+- The `ConfigBuilder` SPI has been extended with a method that allows for a converter with the specified class type to be registered  (link:https://github.com/eclipse/microprofile-config/issues/205[#205]). This change removes the limitation, which was unable to add a lambda converter, from the previous releases.
+
+=== Functional Changes
+
+- Implementations must now support the array converter (link:https://github.com/eclipse/microprofile-config/issues/259[#259]).
+For the array converter, the programmatic lookup of a property (e.g. `config.getValue(myProp, String[].class)`) must support the return type of the array. For the injection lookup, an Array, List or Set must be supported as well (e.g. `@Inject @ConfigProperty(name="myProp") private List<String> propValue;`). 
+
+- Implementations must also support the common sense converters (link:https://github.com/eclipse/microprofile-config/issues/269[#269]) where there is no corresponding type of converters provided for a given class. The implementation must use the class's default instructor taking a string parameter, then try `valueOf(String)` followed by `parse(CharSequence)`. 
```

Comment:
instead of 'default instructor' should we use a constructor with a single `String` argument?

Reply:
well spotted. late night working:o... fixed now.